managed/ui/src/routes.js

```diff
@@ -121,6 +121,10 @@ function validateSession(store, replacePath, callback) {
   const userId = Cookies.get('userId') || localStorage.getItem('userId');
   const customerId = Cookies.get('customerId') || localStorage.getItem('customerId');
   if (_.isEmpty(customerId) || _.isEmpty(userId)) {
+    let location = window.location.pathname;
+    if (location !== '/') {
```

Comment:
I'd add check for `/login` to avoid possible circular redirects from `/login` to `/login`

Reply:
I didn't see that while testing, but there is no harm in having that check so, done